src/test/java/io/jenkins/plugins/ml/IPythonKernelInterpreterTest.java

```diff
@@ -0,0 +1,26 @@
+package io.jenkins.plugins.ml;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class IPythonKernelInterpreterTest {
+
+    /**
+     * Test for {@link IPythonKernelInterpreter}
+     */
+    private KernelInterpreter kernelInterpreter;
+
+    @Before
+    public void setup(){
+        kernelInterpreter = new IPythonKernelInterpreter();
+    }
+
+    @Test
+    public void testToString() {
+        if(kernelInterpreter instanceof IPythonKernelInterpreter){
+            Assert.assertEquals("IPython is not properly initiated","IPython Interpreter",kernelInterpreter.toString());
+        }
```

Comment:
Instead we can just cast here? i.e. if the object is not an instance of `IPythonKernelInterpreter`, that would mean this test wouldn't have any assertion. It's better that every test has at least 1 assertion (or error/failure).

Reply:
I will  add a assertion below.